Sources/OpenPass/Data/OpenPassTokens.swift

```diff
@@ -28,38 +28,134 @@ import Foundation
 
 /// Data object for OpenPass ID and Access Tokens
 @available(iOS 13.0, tvOS 16.0, *)
-public struct OpenPassTokens: Codable {
-    
+public struct OpenPassTokens: Hashable, Codable {
+
     /// ID Token constructed via `idTokenJWT`
     public let idToken: IDToken?
     
     /// ID token as JWT
     public let idTokenJWT: String
 
+    /// Seconds until the ID Token expires
+    public let idTokenExpiresIn: Int64?
+
     /// Access Token
     public let accessToken: String
 
     /// Type of Access Token
     public let tokenType: String
 
-    /// Expiration Time of Token in UTC
+    /// Seconds until the Access Token expires
     public let expiresIn: Int64
+
+    /// Refresh Token
+    public let refreshToken: String?
```

Comment:
I may be missing it, but do you have unit tests which cover the scenario where a previously persisted set of tokens would not have this additional data? Do these automatically just come back as `nil`?